utils/gigaspeech_download.sh

```diff
@@ -40,7 +41,8 @@ fi
 
 if [ $stage -le 2 ]; then
   # Verify the data size or md5.
-  echo "$0: Please add scripts for checking data"
+  echo "$0: checking data consistency(through md5) of downloaded audios"
```

Comment:
To be consistent with other logs, let's use "Checking" instead of "checking"?

consistency(through md5) --> consistency (through md5)

Let's use "audio files" instead of "audios"